docs/src/main/tut/client.md

```diff
@@ -30,6 +30,15 @@ A good default choice is the `PooledHttp1Client`.  As the name
 implies, the `PooledHttp1Client` maintains a connection pool and
 speaks HTTP 1.x.
 
+First, we'll need a dependency on the http4s blaze-client
```

Comment:
I did, but I'm not against putting it in. Probably should include it just to be safe.